ONSdigital/eq-terraform
# RDS Instance

**_What**_
1. Creating an MutliAZ HA database instance
2. Creating a new subnet and renaming the WAF one (so we have two database subnets)
3. Constructing the DATABASE URL and configuring the Survey Runner to use SERVER SIDE STORAGE.

**_How to test**_
1. Check out this branch
2. Modify the deploy_surveyrunner.sh script and point it at 'rds-integration'
3. Deploy a terraform environment
4. Check you can complete the survey
5. Check you can save the survey and come back to it later

**_Who can review**_
Anyone apart from @warrenbailey
